src/Kernel/Character.class.st

```diff
@@ -797,6 +797,14 @@ Character >> isUppercase [
 
 ]
 
+{ #category : #testing }
+Character >> isDoubleQuote [
+	"Returns whether the receiver is a Double Quote"
+
+	| in |
+	^ (in := self asInteger) == 34
```

Comment:
There is no need of a temporary variable here.
That would simplify the code like

```smalltalk
^ (self asInteger) == 34
```

Also remember message precedence: unary methods are always executed before binary methods, and those are before keyword methods. This means that in this case, parenthesis are not required. That would give us

```smalltalk
^ self asInteger == 34
```

Finally, to compare Characters, you don't need to extract their internal code point value. Instead, Characters can be expressed as literal objects. For example, the character `a` is  `$a`, the character `4` is  `$4` (this is kinda the same as single quotes in C. This means you could obtain a literal double quote character by doing `$"`, and with that we could just compare characters!

```smalltalk
^ self == $"
```

Finally, remember in most object oriented languages there are two equality operations: object equality (`=` in Pharo, `.equals()` in Java) and object identity (`==` in both Pharo and Java). Object equality checks if two objects have the same value. Object identity checks if they are **the same object**. In general, we should not use object identity unless really required.
Then I'd simplify further the code to:

```smalltalk
^ self = $"
```